build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -418,71 +222,27 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
             }
 
             // cleanup compilers project output we don't need
-            DeleteMainPublishOutput(outputDir, "compilers");
+            PublishMutationUtilties.CleanPublishOutput(sdkOutputDirectory, "compilers");
             File.Delete(compilersDeps);
-            File.Delete(compilersRuntimeConfig);
```

Comment:
I thought about this, but the deps file is not needing to be deleted in the other place it is used. I suppose I could make it optional.